panel/_dataclasses/pydantic.py

```diff
@@ -0,0 +1,62 @@
+import json
+
+from asyncio import sleep
+from typing import TYPE_CHECKING, Any, Iterable
+
+from param.reactive import bind
+
+from ..pane.markup import JSON
+from .base import ModelUtils
+
+if TYPE_CHECKING:
+    try:
+        from pydantic import BaseModel
+    except ModuleNotFoundError:
+        BaseModel = Any
+else:
+    BaseModel = Any
+
+
+class PydanticUtils(ModelUtils):
+    can_observe_field = False
+    supports_constant_fields = False
+
+    # Todo: Fix issue and remove this hack
+    _is_testing = False
+
+    @classmethod
+    def get_field_names(cls, model: BaseModel) -> Iterable[str]:
+        return tuple(model.model_fields)
+
+    @classmethod
+    def observe_field(
+        cls,
+        model,
+        field: str,
+        handle_change,
+    ):
+        # We don't know if this is possible
+        # Maybe solution can be found in
+        # https://github.com/pydantic/pydantic/discussions/7127 or
```

Comment:
Pretty certain it isn't without significantly modifying the original class.

Reply:
psygnal is a separate project. But it adds observer pattern/ events to Pydantic.